### DISCUSSION TITLE
ffmpeg error 11

### DISCUSSION THREAD

**Discussion by cwebbtw** [Rapptz/discord.py#8436] [Q&A]
Howdi folk,

I'm seeing the following error, but a quick google isn't really showing me what error code -11 is.

`2022-09-05T09:04:12.266259+00:00 app[worker.1]: [2022-09-05 09:04:12] [INFO    ] discord.player: ffmpeg process 52 successfully terminated with return code of -11.`

Anyone know what this error code is?

The error is triggered by the following:

`voice.play(discord.PCMVolumeTransformer(discord.FFmpegPCMAudio(url, **ffmpeg_options))`

**Comment by cwebbtw** [comment]
Debug logs:

> 2022-09-05T11:33:34.308176+00:00 app[worker.1]: voice client play: <bot_music.PCMVolumeTransformer object at 0x7f7424543310>
> 2022-09-05T11:33:34.313409+00:00 app[worker.1]: [2022-09-05 11:33:34] [DEBUG   ] discord.gateway: Sending voice websocket frame: {'op': 5, 'd': {'speaking': 1, 'delay': 0}}.
> 2022-09-05T11:33:34.326062+00:00 app[worker.1]: [2022-09-05 11:33:34] [DEBUG   ] discord.player: Preparing to terminate ffmpeg process 61.
> 2022-09-05T11:33:34.327058+00:00 app[worker.1]: [2022-09-05 11:33:34] [INFO    ] discord.player: ffmpeg process 61 successfully terminated with return code of -11.
> 2022-09-05T11:33:34.327430+00:00 app[worker.1]: [2022-09-05 11:33:34] [DEBUG   ] discord.gateway: Sending voice websocket frame: {'op': 5, 'd': {'speaking': 0, 'delay': 0}}.
> 2022-09-05T11:33:36.021390+00:00 app[worker.1]: [2022-09-05 11:33:36] [DEBUG   ] discord.gateway: Keeping shard ID None voice websocket alive with timestamp 1662377616021.
> 2022-09-05T11:33:36.026241+00:00 app[worker.1]: [2022-09-05 11:33:36] [DEBUG   ] discord.gateway: Sending voice websocket frame: {'op': 3, 'd': 1662377616021}.
> 2022-09-05T11:33:36.051433+00:00 app[worker.1]: [2022-09-05 11:33:36] [DEBUG   ] discord.gateway: Voice websocket frame received: {'op': 6, 'd': 1662377616021}

**Reply by iasolis** [reply]
I have the same problem when trying to run a music chatbot on Heroku, if you found a solution, could you share it?

**Comment by Kimsec** [comment]
> Howdi folk,
> 
> I'm seeing the following error, but a quick google isn't really showing me what error code -11 is.
> 
> `2022-09-05T09:04:12.266259+00:00 app[worker.1]: [2022-09-05 09:04:12] [INFO ] discord.player: ffmpeg process 52 successfully terminated with return code of -11.`
> 
> Anyone know what this error code is?
> 
> The error is triggered by the following:
> 
> `voice.play(discord.PCMVolumeTransformer(discord.FFmpegPCMAudio(url, **ffmpeg_options))`

have the same issue now, have not yet found a solution.. this happened after I upgraded my system to the newest DSM (Synology).. Any tips would be highly appreciated.

**Comment by arimanyus** [comment]
Old issue but after spending two weeks trying to figure out why my video generation function on AWS Lambda wasn't working (ffmpeg was returning -11 everytime). I found out that there are some copyright issues with avc1 and certain ffmpeg installations won't have it. Using mp4v instead worked for me.
